wp-content/themes/core/assets/pcss/color/_variables.pcss

```diff
@@ -13,69 +13,65 @@
 	 * Color Naming Site: https://chir.ag/projects/name-that-color/
 	 * ----------------------------------------------------------------------- */
 
-	--color-royal-blue: var(--wp--preset--color--royal-blue);
-	--color-jacksons-purple: var(--wp--preset--color--jacksons-purple);
+	--color-blue: var(--wp--preset--color--blue, #3050e5);
+	--color-blue-rgb: 48, 80, 229;
```

Comment:
Couldn't the blue preset value be the rgb() color?

Reply:
I wasn't able to find a combination of this that worked. It doesn't appear you can just assign `48, 80, 229` to the preset value. It expects a prefix like `#` or `rgb(`. Setting it to `rgb(48, 80, 229)` doesn't allow you to use in an `rgba()` definition. 

Since this color is only used for the `::selection` code, I could also just one-off the `rgba()` definition to the selection code instead?

Reply:
Let's keep it defined as a color var, but maybe name it specifically like `--color-selection-highlight` or something similar?

Reply:
Updated to new naming for variable; Moved it to it's own section in the colors file; Added a description for this new section and updated the value to use a #RRGGBBAA format.